warehouse/static/sass/settings/_fonts.scss

```diff
@@ -14,7 +14,7 @@
 
 // SETTINGS - FONTS
 
-$base-font-family: "Source Sans Pro", "Helvetica", Arial, sans-serif;
+$base-font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Helvetica, Arial, sans-serif, Apple Color Emoji, Segoe UI Emoji;
 $code-font: "Source Code Pro", monospace;
```

Comment:
Unrelated but I think we also need to consider something like `Fira Code` here (separate discussion needed)

Reply:
You mean a "system font stack" for the monospaced case?

Maybe https://github.com/pradyunsg/furo/blob/ecfee9c5a3872cfac72f5ee8c63c7d474ee8e60f/src/furo/assets/styles/variables/_index.scss#L41-L42 would work?

Reply:
Probably. But it's out of the scope of this PR.

Reply:
Please don’t use “BlinkMacSystemFont”; just using “system-ui” is enough in Blink-based browsers.